backend/tests/test_stub.py

```diff
@@ -58,7 +58,7 @@ def test_test_endpoint_invalid_input():
 
 @pytest.mark.asyncio
 async def test_books_add_endpoint(book_dict):
-    await TitleMetadata.objects.delete(each=True)
+    await TitleMetadata.objects.delete(title="wikipedia_lg_all")
```

Comment:
Either retrieve this string from the book_dict or create a fixture for this ident so we don't have this static string all over the tests.